builder/gen-dockerfile/contents/src/data/Dockerfile

```diff
@@ -0,0 +1,3 @@
+# Dockerfile extending the generic Node image with application files for a
+# single application.
+FROM gcr.io/google_appengine/nodejs
```

Comment:
Oh, good catch–yes, it should. See the internal instructions doc–there's a note under "Write a Cloud Build specification"